iOS/Flows/Conversation/Expressions/ExpressionViewController.swift

```diff
@@ -123,6 +132,22 @@ class ExpressionViewController: ViewController {
         
         self.didCompleteExpression?(saved)
     }
+
+    private func createVideoExpression() async {
+        guard let videoURL = self.videoURL else { return }
+
+        let videoData = try! Data(contentsOf: videoURL)
```

Comment:
Ideally this would be async so the UI doesn't freeze when writing the video data.